include/stdlib.h

```diff
@@ -147,15 +146,7 @@ int       on_exit(CODE void (*func)(int, FAR void *), FAR void *arg);
 /* _Exit() is a stdlib.h equivalent to the unistd.h _exit() function */
 
 void      _exit(int status); /* See unistd.h */
```

Comment:
remove this copy of _exit prototype. it's only for the _Exit macro.

Reply:
No, we can't, because all exit/_exit/_Exit specify in the standard:
https://linux.die.net/man/3/_exit
https://en.cppreference.com/w/c/program/exit
https://en.cppreference.com/w/c/program/_Exit

Reply:
in the standards, _exit is in unistd.h, not stdlib.h.
we have an _exit prototype in unistd.h.
this prototype in stdlib.h is no longer necessary.

Reply:
Ok, I misunderstand your comment, fix.